README.md

```diff
@@ -87,9 +87,31 @@ These are the commands available to use with Action Scheduler:
 
 The best way to get a full list of commands and their available options is to use WP CLI itself. This can be done by running `wp action-scheduler` to list all Action Scheduler commands, or by including the `--help` flag with any of the individual commands. This will provide all relevant parameters and flags for the command.
 
-### Improving Performance with `--group`
+### Cautionary Note on Action Dependencies when using `--group` or `--hooks` Options
 
-Being able to run queues for specific groups of actions is valuable at scale. Why? Because it means you can restrict the concurrency for similar actions.
+The `--group` and `--hooks` options should be used with caution if you have an implicit dependency between scheduled actions based on their schedule.
+
+For example, consider two scheduled actions for the same subscription:
+
+* `scheduled_payment` scheduled for `2015-11-13 00:00:00` and
+* `scheduled_expiration` scheduled for `2015-11-13 00:01:00`.
+
+Under normal conditions, Action Scheduler will ensure the `scheduled_payment` action is run before the `scheduled_expiration` action. Becuase that's how they are scheduled.
+
+However, if using the `--hooks` option, and processing payment actions in a separate queue to expirations action, this dependency is not guaranteed.
+
+For example, consider the case of:
+
+* 100,000 `scheduled_payment` actions all for `2015-11-13 00:00:00`, and
```

Comment:
There should be a comma after "actions", and you can omit `, and` from the end.

Reply:
> you can omit `, and` from the end

Removing `and` introduces ambiguity - _are both of these sets of actions required for the example? Or only one or them? Or can there be either both or just one?_

I've removed the `and` but addressed the ambiguity by changing the leading sentence to read:

> For example, consider a site with both:

I think that's better, but it also means [it's even more important this information isn't missed](5 days ago).